content/language/word-list.md

```diff
@@ -10,17 +10,27 @@ Lists which permit or deny a set of nouns, or select enabled features.
 
 **Proposed alternative:** 
 
-* allowlist/denylist 
-* allowedNouns/deniedNouns
+As a verb:
+* For whitelist: allow, accept, enable, include
+* For blacklist: deny, block, disable, exclude, skip, ignore
+
+As an adjective:
+* For whitelisted: allowed, accepted, enabled, included, valid
+* For blacklisted: denied, blocked, disabled, excluded, skipped, ignored, buggy, invalid
+
+As a noun:
+* Use the adjective as an entity prefix (e.g., _allowedNouns_)
+* Also acceptable: allowlist, denylist, etc.
```

Comment:
Same comment as below – the Language workstream is re-evaluating these suggestions and will revise the recommendations themselves shortly. I like your revisions to the reasoning itself and would like to merge those, however.